src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Remote/IRemoteAutoInsertService.cs

```diff
@@ -3,22 +3,20 @@
 
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.ExternalAccess.Razor;
 using Microsoft.CodeAnalysis.Text;
 
 using Response = Microsoft.CodeAnalysis.Razor.Remote.RemoteResponse<Microsoft.CodeAnalysis.Razor.Protocol.AutoInsert.RemoteAutoInsertTextEdit?>;
 
+namespace Microsoft.CodeAnalysis.Razor.Remote;
+
 internal interface IRemoteAutoInsertService
 {
     ValueTask<Response> GetAutoInsertTextEditAsync(
         RazorPinnedSolutionInfoWrapper solutionInfo,
         DocumentId documentId,
         LinePosition position,
         string character,
-        bool autoCloseTags,
-        bool formatOnType,
-        bool indentWithTabs,
-        int indentSize,
+        RemoteAutoInsertOptions options,
```

Comment:
Great idea to consolidate these! The updates to `RemoteAutoInsertService` that support this options type look particularly nice.